test/resources/scheme/PureScheme.Test.Binder.Object/lib.sls

```diff
@@ -10,13 +10,13 @@
       (v)
       (scm:cond
         ((scm:and
-            (scm:= (scm:hashtable-ref v "a" (scm:error #f "Key not found")) 1)
-            (scm:= (scm:hashtable-ref v "b" (scm:error #f "Key not found")) 2)
-            (scm:= (scm:hashtable-ref v "c" (scm:error #f "Key not found")) 3))
+            (scm:= (scm:hashtable-ref v "a" (scm:quote "Key not found: a")) 1)
```

Comment:
Quoting a string makes no sense.

Reply:
Can we also print `v` here?

Reply:
Unfortunately the Chez's built-in printer for hashtables doesn't look inside the structure and it just prints `#<hashtable>` so I don't think it would be useful.